src/index.ts

```diff
@@ -119,6 +119,42 @@ export const compare = memoize((versionA: string | null, versionB: string | null
 }, (a: string, b: string) => `${a} && ${b}`);
 
 /**
+ * @summary Compare order of versions in reverse
+ * @name rcompare
+ * @public
+ * @function
+ *
+ * @description The reverse of `.compare()`. Accepts string or null values and compares
+ * them, returning a number indicating sort order. Values are parsed for valid semver
+ * strings. Sorts an array of versions in descending order when passed to `Array.sort()`.
+ *
+ * @param {string|null} versionA - The first version to compare
+ * @param {string|null} versionB - The second version to compare
+ *
+ * @returns {number} Returns `0` if `versionA == versionB`,
+ * or `-1` if `versionA` is greater, or `1` if `versionB` is greater.
+ * null values are always weighted above string values, and string values are always
+ * weighted above valid semver values.
```

Comment:
You've flipped the order of this description (B > A, instead of A > B), and also the ordering of these comparisons (null > string, instead of string > null). I think that makes this 2nd sentence here wrong, as it's kind of become a double negative.

In practice, `rcompare(null, 'a string')` and `rcompare('a string', '1.2.3')` return 1, and this description says the opposite.